<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QMainWindow" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>803</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>北京同明眼科专家</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../source/timg.jpg</normaloff>../source/timg.jpg</iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;楷体&quot;;
background-color: rgb(255, 255, 255);
gridline-color: rgb(255, 0, 0);
font: 12pt &quot;黑体&quot;;
selection-color: qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:0.077, fx:0.5, fy:0.5, stop:0 rgba(0, 169, 255, 147), stop:0.497326 rgba(0, 0, 0, 147), stop:1 rgba(0, 169, 255, 147));
selection-color: rgb(255, 0, 0);</string>
  </property>
  <property name="iconSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_user_hospital">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Agency FB&quot;;
background-color: rgba(46, 187, 150, 125);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;北京大学第三人民医院&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_user_name">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Agency FB&quot;;
background-color: rgba(46, 187, 150, 125);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#ffffff;&quot;&gt;杨雷&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_datetime">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;仿宋&quot;;
background-color: rgba(46, 187, 150, 125);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;08:08:56 2017-11-23&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_head">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../../myself/项目/朱总(北京同明眼科)/png/head.png</pixmap>
    </property>
    <property name="margin">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_user_title">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Agency FB&quot;;
background-color: rgba(46, 187, 150, 125);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#ffffff;&quot;&gt;主任医师&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_weekday">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;仿宋&quot;;
background-color: rgba(46, 187, 150, 125);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;星期一&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>130</y>
      <width>1211</width>
      <height>671</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>671</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-color: rgb(0, 170, 0);
selection-background-color: rgb(0, 170, 0);
font: 16pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>待诊</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>41</y>
        <width>1201</width>
        <height>591</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_search">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>0</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(0, 85, 255);
font: 9pt &quot;Agency FB&quot;;
border-color: rgb(30, 192, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_search">
      <property name="geometry">
       <rect>
        <x>1100</x>
        <y>0</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>搜  索</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>工作台</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>191</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>新生儿（3--7天）眼保健情况记录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>40</y>
        <width>191</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>42天眼保健情况记录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>40</y>
        <width>191</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>5--6个月眼保健情况记录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>40</y>
        <width>191</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>1岁儿童眼保健情况记录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>230</y>
        <width>191</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>2岁儿童眼保健情况记录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>230</y>
        <width>191</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>3岁儿童眼保健情况记录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>230</y>
        <width>191</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>3岁儿童眼保健情况记录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>230</y>
        <width>191</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>4岁儿童眼保健情况记录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>430</y>
        <width>191</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>5岁儿童眼保健情况记录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_12">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>430</y>
        <width>191</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>6岁儿童眼保健情况记录</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>档案馆</string>
     </attribute>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>1201</width>
        <height>591</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_search_2">
      <property name="geometry">
       <rect>
        <x>1110</x>
        <y>0</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>搜  索</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_search_2">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>0</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(0, 85, 255);
font: 9pt &quot;Agency FB&quot;;
border-color: rgb(30, 192, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_23">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>191</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>杨同学</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_24">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>80</y>
        <width>191</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>张同学</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_25">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>80</y>
        <width>191</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>刘同学</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_26">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>80</y>
        <width>191</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>李同学</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>统计</string>
     </attribute>
     <widget class="QLabel" name="label_head_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>150</y>
        <width>191</width>
        <height>201</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../../myself/项目/朱总(北京同明眼科)/png/statistics.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_head_3">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>150</y>
        <width>191</width>
        <height>201</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../../myself/项目/朱总(北京同明眼科)/png/statistics.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_head_4">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>150</y>
        <width>191</width>
        <height>201</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../../myself/项目/朱总(北京同明眼科)/png/statistics.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1201</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 187, 150);
border-color: rgb(255, 255, 255);
gridline-color: rgb(255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>pushButton</zorder>
   <zorder>label_user_hospital</zorder>
   <zorder>label_user_name</zorder>
   <zorder>label_datetime</zorder>
   <zorder>label_head</zorder>
   <zorder>label_user_title</zorder>
   <zorder>label_weekday</zorder>
   <zorder>tabWidget</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
